src/components/navbar.tsx

```diff
@@ -1,18 +1,64 @@
+"use client";
 import styled from "@emotion/styled";
 import Link from "next/link";
 import { usePathname } from "next/navigation";
+import chatIcon from "../../public/svg/icon-chat.svg";
+import burgerIcon from "../../public/svg/icon-burger.svg";
+import closeIcon from "../../public/svg/icon-close.svg";
+import Image from "next/image";
+import { useEffect, useState } from "react";
 
 const StyledNav = styled.nav`
```

Comment:
<img width="506" alt="Screenshot 2024-02-16 at 15 03 16" src="https://github.com/web-space-dev/website-frontend/assets/29145846/42bbc86c-d7d6-4dfd-927e-b843e31246a7">

Most of the website the nav will be over dark backgrounds, dont worry about the functionality for handling this, for now you can just pass a prop saying dark="true", but if you can match the styles in the figma for that, that would be great

Reply:
![Screenshot 2024-02-17 at 13 06 08](https://github.com/web-space-dev/website-frontend/assets/147092788/6b7204ea-a183-4035-9503-f9c04e8d066e)


I have some difficulty to change the SVG stroke, so I temporary changed the SVG to white without any props.

Reply:
Yes this is something I'm running into also right now, I'm going to try look for a solution for the SVG strokes.

I just noticed two small things:
1. Could you change the active nav item border colour to also be white
2. On the project page, the nav menu seems to be loaded twice on top of itself

Reply:
done for 1 and 2